README.rst

```diff
@@ -6,17 +6,10 @@
 Cirq is a Python library for writing, manipulating, and optimizing quantum
 circuits and running them against quantum computers and simulators.
```

Comment:
What's the reason for these changes? Why did you re link the image to your repo?

Reply:
i just wanted to have a quick link to the travis-ci build test. 
i did not wanted to include this patch in the pull request.